src/api/env-variables.ts

```diff
@@ -1,23 +1,25 @@
 export interface EnvVariables {
-  [key: string]: string | number | boolean;
+  [key: string]: { value: string | number | boolean, secure: boolean };
```

Comment:
you can put this in two lines, it would be nicer:
```ts
[key: string]: {
  value: string | number | boolean;
  secure: boolean;
}
```

Reply:
done